keyboards/handwired/symmetric70_proto/readme.md

```diff
@@ -0,0 +1,20 @@
+# symmetric70_proto
+
+![symmetric70_proto](https://i.imgur.com/Br4pH9o.jpg)
+<img width="90%" alt="74HC157" src="https://user-images.githubusercontent.com/2170248/97360929-33765280-18e2-11eb-9eeb-791133e4d328.PNG">
```

Comment:
If you host this on imgur and add `l` to the filename (as above), you won't need the img tag here.

Reply:
Thanks.